### PR TITLE
Added to node configuration file re: deploying an instance of a given…

### PR DESCRIPTION
**Pull request by tpoitras**
… pod on each node

RE: docs trello card https://trello.com/c/CEcIyJwe

PING @abhgupta for tech review

do you think it needs more than this?

Also, should we provide a sample pod manifest file? How is the admin expected to know how to make one?

### CODE REVIEW

**Comment by tpoitras** [comment]
PING @abhgupta again. :ear:

File: admin_guide/master_node_configuration.adoc [hunk 1]
```diff
@@ -268,7 +268,9 @@ kind: NodeConfig
 masterKubeConfig: node.kubeconfig
 networkPluginName: ""
 nodeName: node1.example.com
-podManifestConfig: null
+podManifestConfig: <1>
+  path: "/tmp/path/to/pod-manifest-file" <2>
```
Comment (abhgupta) [inline admin_guide/master_node_configuration.adoc]: I would remove /tmp from the path - the implications of that could be misleading.
`/path/to/pod-manifest-file` should be adequate.

**Comment by abhgupta** [comment]
The pod manifest is the same pod manifest file that a user would use for creating a pod via the CLI. Don't think we need an example for that, though mentioning this fact would be helpful.

**Comment by abhgupta** [comment]
Other than the comment, LGTM

**Comment by adellape** [comment]
@tpoitras If you want to show an example pod manifest, you could link to https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/pods_and_services.html#pods (the Pod Object Definition example).

**Comment by tpoitras** [comment]
Updated! Thanks @abhgupta & @adellape -- should be alright to merge now. :zap:

**Comment by adellape** [comment]
LGTM

**Pull request merged**